<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParaUnitSetupDialog</class>
 <widget class="QDialog" name="ParaUnitSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParaUnitSetupDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbGroup">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>15</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
